<?xml version="1.0"?>
<launch>
  <!-- Debug Info -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />
  <arg name="planning_plugin" default="ompl_interface/OMPLPlanner" />
  <arg name="planning_adapters" default="default_planner_request_adapters/AddTimeParameterization default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints" />
  <arg name="robot_interface_plugin" />

  <node name="workspace_tool" pkg="rcprg_planner" type="workspace_tool" respawn="false" launch-prefix="$(arg launch_prefix)" output="screen" >
    <param name="planning_plugin" value="$(arg planning_plugin)" />
    <param name="request_adapters" value="$(arg planning_adapters)" />
    <param name="start_state_max_bounds_error" value="0.1" />
    <param name="max_sampling_attempts" value="1000" />
    <param name="robot_interface_plugin" value="$(arg robot_interface_plugin)" />
    <rosparam command="load" file="$(find rcprg_planner)/config/ompl_planning.yaml"/>
  </node>
</launch>
